modules/core/src/main/java/org/apache/ignite/internal/managers/systemview/GridSystemViewManager.java

```diff
@@ -53,6 +57,9 @@
  */
 public class GridSystemViewManager extends GridManagerAdapter<SystemViewExporterSpi>
     implements ReadOnlySystemViewRegistry {
+    /** Class name for a SQL view exporter of system views. */
+    public static final String SYSTEM_VIEW_SQL_SPI = "org.apache.ignite.internal.managers.systemview.SqlViewExporterSpi";
```

Comment:
Visibility scope for this constant is too broad.

Reply:
fixed.